docs/content/glossary.md

```diff
@@ -39,6 +40,18 @@ The [Installation] custom resource represents an installation of a bundle in Por
 
 [Installation]: /operator/file-formats/#installation
 
+### CredentialSet
+
+The [CredentialSet] custom resource represents a credential set in Porter.
+
+[CredentialSet]: /operator/file-formats/#credentialset
+
+A CredentialSet supports a credential source of `secret` for Porter secrets plugins. Secrets 
+must be configured based on the secret [plugin type](/plugins). 
```

Comment:
```suggestion
A CredentialSet supports a credential source of `secret` for Porter secrets plugins. Secrets 
source keys may vary depending on which [secret plugin](/plugins/) you have configured. The [host secrets plugin](/plugins/host/) is not a good fit for use with the Porter Operator because environment variables, files are not a recommended way to manage secrets on a cluster. The [kubernetes.secrets plugin](https://release-v1.porter.sh/plugins/kubernetes/#secrets) can retrieve secrets from native Kubernetes secrets, and otherwise we recommend that an external secret store such as [Azure KeyVault](/plugins/azure/#secrets) or [Hashicorp Vault](/plugins/hashicorp/) are configured instead.
```